### PR TITLE
ap-node-modes

### PR DESCRIPTION
**Pull request by AndrePanin**
Added dev net and multi node local testnet.

### CODE REVIEW

File: docs/node/multi-node.md [hunk 1]
```diff
@@ -5,4 +5,38 @@ sidebar_position: 5
...
+  --chain=local
+  --bob
+  --port 30334
+  --ws-port 9945
+  --telemetry-url 'ws://telemetry-backend-shard.gear-tech.io:32001/submit 0'
+  --name 'bob'
+  --validator
+```
+
+3. Execute `cargo run -- --help` to learn more about the template node's CLI options.
```
Comment (shamilsan) [inline docs/node/multi-node.md:42]: This section seems to be unrelated to multi-node, isn't it?

Comment (AndrePanin) [reply docs/node/multi-node.md]: ok, removed

**Review by shamilsan — APPROVED** [review]
